### PR TITLE
Update configs for distro

### PR DESCRIPTION
**Pull request by anthonydresser**

### CODE REVIEW

File: build/npm/update-distro.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ const path = require('path');
 const fs = require('fs');
 
 const rootPath = path.dirname(path.dirname(path.dirname(__dirname)));
-const vscodePath = path.join(rootPath, 'vscode');
-const distroPath = path.join(rootPath, 'vscode-distro');
+const vscodePath = path.join(rootPath, 'azuredatastudio');
```
Comment (Charles-Gagnon) [inline build/npm/update-distro.js]: Don't need edit comments for this file?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.14.0",
-  "distro": "01d4471bbc2227c73907d5b42d6a54259e88b77c",
+  "distro": "08714f9db0e9c297aa44256ea74b708d5ec5ae05",
```
Comment (Charles-Gagnon) [inline package.json]: For my own knowledge, what does this hash refer to?

File: product.json [hunk 3]
```diff
@@ -23,65 +23,5 @@
 	"privacyStatementUrl": "https://privacy.microsoft.com/en-us/privacystatement",
 	"telemetryOptOutUrl": "https://github.com/Microsoft/azuredatastudio/wiki/How-to-Disable-Telemetry-Reporting",
 	"urlProtocol": "azuredatastudio",
-	"enableTelemetry": true,
-	"aiConfig": {
-		"asimovKey": "AIF-444c3af9-8e69-4462-ab49-4191e6ad1916"
-	},
-	"sendASmile": {
-		"reportIssueUrl": "https://github.com/Microsoft/azuredatastudio/issues/new?labels=customer%20reported%20issue",
```
Comment (Charles-Gagnon) [inline product.json]: Are these being injected somewhere else then? If not why are you taking them out?

Comment (anthonydresser) [reply product.json]: This entire file is replaced.

File: package.json (hunk 2 above)
Comment (anthonydresser) [reply package.json]: What commit to merge from.

File: product.json (hunk 3 above)
Comment (Charles-Gagnon) [reply product.json]: Replaced during build time or something? Where does this config live now then?

Comment (anthonydresser) [reply product.json]: Same repo as the icons, replaced as part of the same mixin

**Pull request merged**